src/shared/components/namespaceColumns/NumericNamespaceColumnFormatter.tsx

```diff
@@ -1,14 +1,17 @@
 import * as React from 'react';
-import { Mutation } from 'cbioportal-ts-api-client';
 import _ from 'lodash';
 import generalStyles from 'shared/components/mutationTable/column/styles.module.scss';
 
+export type WithNamespaceColumns = {
```

Comment:
this name doesn't totaly make sense to me.  like, what with namespace columns?  what is the point of the wrapper object if it only has one node called namespaceColumns?

Reply:
It marks types with a namespaceColumns property. 
Mutation, CNA and structural variant types all can have namespaceColumns. 
Another option would be to list all those types (and any future namespaceColumn-types)? 
I thought it would be more readable.